gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/medication/MedicationRequestPlanMapper.java

```diff
@@ -187,11 +181,15 @@ private String extractTermText(RCMRMT030101UKDiscontinue discontinue) {
 
         var stringBuilder = new StringBuilder();
```

Comment:
Could we name the stringBuilder variable to be more in line with the method name and the expected result?